src/views/forms/StakeForm/index.tsx

```diff
@@ -187,6 +184,23 @@ export const StakeForm = ({ onDone, className }: StakeFormProps) => {
           isLoading={isLoading || Boolean(isAmountValid && !fee)}
           amount={amount}
         />
+        <$LegalDisclaimer>
+          {stringGetter({
+            key: STRING_KEYS.STAKING_LEGAL_DISCLAIMER,
+            params: {
+              KEPLR_DASHBOARD_LINK: (
+                <$Link withIcon onClick={openKeplrDialog}>
+                  {stringGetter({ key: STRING_KEYS.KEPLR_DASHBOARD })}
+                </$Link>
+              ),
+              STRIDE_LINK: (
+                <$Link withIcon onClick={openStrideDialog}>
+                  Stride
+                </$Link>
```

Comment:
currently clicking navigate to keplr dashboard does nothing because probably it's not "forceOpenDialog"
<img width="602" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/7229bb69-b2b4-4b89-9e18-a2464e40b96d">

Reply:
same with stride!

Reply:
oooh i didn't go beyond the initial shift - tyty will update